### ISSUE TITLE
Multiple statements not supported in MySQL init

### ISSUE THREAD

**Issue by lyndsysimon** [yogthos/migratus#93]
The following `init.sql` fails for MySQL:

```sql
create table foo (id int not null auto_increment primary key);
--;;
create table baz (id int not null auto_increment primary key);
```

Error message:

`com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '--;;`

**Comment by lyndsysimon** [comment]
The same file does work as a migration, however.

**Comment by yogthos** [comment]
Yeah, it looks like the init script is [not being parsed](https://github.com/yogthos/migratus/blob/12356ebf68193db13d69cf99302d8202b504359c/src/migratus/database.clj#L289) the way regular migrations are. Technically, it's only meant for schema initialization, so table creation should be happening as a regular migration. It would probably make sense to parse the script the same way migrations are though.

**Comment by WorldsEndless** [comment]
This is a major concern for my team, too; the fact that we cannot include multiple lines in the init is a major disparity between Migratus  for Postgres (which works great!) and Migratus for MySQL.

**Comment by WorldsEndless** [comment]
Further investigation verifies that migratus/reset also breaks for MySQL when there are multiple SQL statements in a file that is not init.sql.

**Comment by ds21194** [comment]
anything new regarding this? 🙏🏻

**Comment by lotuc** [comment]
Just encountered the problem, and found that `init` run the init script with `next.jdbc/execute!`. Its multiple statement execution is database-dependent.

Just did a dirty patch for my MySQL database initialization:

```Clojure
(require '[migratus.core :as migratus])
(require '[migratus.database])
(require '[migratus.migration.sql])

(with-redefs [migratus.database/run-init-script!
                (fn [_init-script-name init-script conn modify-sql-fn _transaction?]
                  (try
                    (doseq [s (migratus.migration.sql/split-commands init-script nil)]
                      (jdbc/execute! conn (modify-sql-fn s)))
                    (catch Throwable t
                      (throw t))))]
    (migratus/init config)))
```
